packages/path_provider/path_provider_platform_interface/lib/path_provider_platform_interface.dart

```diff
@@ -62,6 +62,12 @@ abstract class PathProviderPlatform extends PlatformInterface {
         'getApplicationDocumentsPath() has not been implemented.');
   }
 
+  /// Path to a directory where application specific cache data can be stored.
+  Future<String?> getApplicationCachePath() {
```

Comment:
There should also be a unit test that this throws; we've not been good about testing this default behavior traditionally, but we've been adding it for new methods.